microservice/challenge-1/app42/docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: "3.3"
```

Comment:
Why not include the frontend service here?

Reply:
Of course. But this file is not completed and I mentioned it in the improvements section.
Since I promised to deliver the code before Monday noon, I did not have enough time to finish this and some other features too.

Reply:
That's fine, just curious =]